openwisp_controller/config/admin.py

```diff
@@ -698,9 +710,55 @@ class Media(BaseConfigAdmin):
         js = list(BaseConfigAdmin.Media.js) + [f'{prefix}js/vpn.js']
 
 
+class DeviceGroupForm(BaseForm):
+    class Meta(BaseForm.Meta):
+        model = DeviceGroup
+        widgets = {'context': DeviceGroupJsonSchemaWidget}
+        labels = {'context': _('Metadata')}
+        help_texts = {
+            'context': _(
+                # TODO: Rephrase this
+                'In this section you can add metadata of the DeviceGroup'
+            )
+        }
+
+
+class DeviceGroupAdmin(BaseAdmin):
```

Comment:
you know what I did forgot in the issue description? Multitenancy.. have you made sure this admin class is multitenant ready?

Reply:
Added automated tests as well. :)